widgetssdk/src/main/java/com/glia/widgets/chat/adapter/ChatAdapter.kt

```diff
@@ -120,50 +116,28 @@ internal class ChatAdapter(
                     getImageFileFromDownloadsUseCase,
                     getImageFileFromNetworkUseCase,
                     schedulers,
-                    uiTheme
+                    uiTheme,
+                    onImageItemClickListener
                 )
             }
 
-            OPERATOR_FILE_VIEW_TYPE -> {
+            OPERATOR_FILE_VIEW_TYPE ->
                 OperatorFileAttachmentViewHolder(
                     ChatAttachmentOperatorFileLayoutBinding.inflate(inflater, parent, false),
-                    uiTheme
+                    uiTheme,
+                    onFileItemClickListener
                 )
-            }
 
-            OPERATOR_MESSAGE_VIEW_TYPE -> {
-                OperatorMessageViewHolder(
-                    ChatOperatorMessageLayoutBinding.inflate(inflater, parent, false),
-                    uiTheme
-                )
-            }
+            OPERATOR_MESSAGE_VIEW_TYPE ->
+                OperatorMessageViewHolder(ChatOperatorMessageLayoutBinding.inflate(inflater, parent, false), uiTheme)
 
-            MEDIA_UPGRADE_ITEM_TYPE -> {
-                MediaUpgradeStartedViewHolder(
-                    ChatMediaUpgradeLayoutBinding.inflate(inflater, parent, false),
-                    uiTheme
-                )
-            }
+            MEDIA_UPGRADE_ITEM_TYPE ->
+                MediaUpgradeStartedViewHolder(ChatMediaUpgradeLayoutBinding.inflate(inflater, parent, false), uiTheme)
```

Comment:
I wanted to ask for a while. Your PRs always have a lot of styling changes. Are these done manually or automatically? What are you using?
We could either agree to use the same tools by everyone in the team. Or agree to do those in separate PRs. I feel like more then third of this commit is styling changes.

Reply:
Android Studio with the default profile: I think it takes formatting from our `.editorconfig` file, so it should be pretty much the same for each of us.

![image](https://github.com/user-attachments/assets/9e557ce8-976c-46b8-8ff3-350351aff007)

Reply:
By the way, I also thought that part of the changes are related to formatting. And there is a chance, these kind of changes might be done in a separate PR. For example, do the coding part, commit, after that go through the changed files and apply automatic formatting, commit separately.

Reply:
This automatic formatting is part of the commit process, so it will not work after committing, and we have to manually walk through the files and apply formatting.

Reply:
Yes, that's why I do it manually. And sometimes miss to auto-format something. 🙂